### PR TITLE
Fixed #226; Environment variables for batch-set limits

### LINKED ISSUE #226: Environment variables for batch-set limits

### PR DESCRIPTION
**Pull request by tilmankamp**
@kdavis-mozilla Please take a look.

### CODE REVIEW

File: DeepSpeech.ipynb [hunk 1]
```diff
@@ -98,6 +98,10 @@
     "ds_importer = os.environ.get('ds_importer', 'ted')\n",
     "ds_dataset_path = os.environ.get('ds_dataset_path', os.path.join('./data', ds_importer))\n",
     "\n",
+    "ds_limit_train = int(os.environ.get('ds_limit_train', 0))\n",
+    "ds_limit_dev   = int(os.environ.get('ds_limit_dev',   0))\n",
+    "ds_limit_test  = int(os.environ.get('ds_limit_test',  0))\n",
```
Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Could you name the python variables, not the environment variables, `limit_train`, `limit_dev` and `limit_test`? This is just to be in line with the naming of the other python variable/environment variable pairs

Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: Could you move these into the **Global Constants** section? Just because most tunable parameters are there.

Comment (kdavis-mozilla) [inline DeepSpeech.ipynb]: After you move these to the **Global Constants** section, could you also document them there like the other **Global Constants** are documented.

**Comment by tilmankamp** [comment]
@kdavis-mozilla Please take another look.

**Comment by kdavis-mozilla** [comment]
LGTM

**Pull request merged**